### ISSUE TITLE
[Regression] [CLI] help advanced fails with a FormatException

### ISSUE THREAD

**Issue by seijikun** [duplicati/duplicati#3136]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info
- **Duplicati version**: 2.0.3.2_canary_2018-03-16
- **Operating system**: Linux
- **Backend**: Hubic

## Description
Running `duplicati-cli help advanced` fails with the following exception, since the last update (from 2.0.2.21_canary_2018-03-06):

```
System.FormatException: Index (zero based) must be greater than or equal to zero and less than the size of the argument list.
  at System.Text.StringBuilder.AppendFormatHelper (System.IFormatProvider provider, System.String format, System.ParamsArray args) [0x000ff] in <f10709aa211442c4867e5ca6f2772c6b>:0 
  at System.String.FormatHelper (System.IFormatProvider provider, System.String format, System.ParamsArray args) [0x00023] in <f10709aa211442c4867e5ca6f2772c6b>:0 
  at System.String.Format (System.String format, System.Object[] args) [0x00020] in <f10709aa211442c4867e5ca6f2772c6b>:0 
  at Duplicati.Library.Localization.MockLocalizationService.Localize (System.String message) [0x00000] in <733aabefd4c742098f2b502149400258>:0 
  at Duplicati.Library.Localization.Short.LC.L (System.String message) [0x00000] in <733aabefd4c742098f2b502149400258>:0 
  at Duplicati.Library.Main.Strings.Options.LogfilelogfiltersLong (System.String delimiter) [0x00000] in <cad7b2bda0054aeb8800ccf4b5d3bb31>:0 
  at Duplicati.Library.Main.Options.get_SupportedCommands () [0x00590] in <cad7b2bda0054aeb8800ccf4b5d3bb31>:0 
  at Duplicati.CommandLine.Help.PrintUsage (System.IO.TextWriter outwriter, System.String topic, System.Collections.Generic.IDictionary`2[TKey,TValue] options) [0x00400] in <4f69d9e98e8444e5abbb41ca4fee555e>:0 
  at Duplicati.CommandLine.Commands.Help (System.IO.TextWriter outwriter, System.Action`1[T] setup, System.Collections.Generic.List`1[T] args, System.Collections.Generic.Dictionary`2[TKey,TValue] options, Duplicati.Library.Utility.IFilter filter) [0x0001f] in <4f69d9e98e8444e5abbb41ca4fee555e>:0 
  at (wrapper delegate-invoke) System.Func`6[System.IO.TextWriter,System.Action`1[Duplicati.Library.Main.Controller],System.Collections.Generic.List`1[System.String],System.Collections.Generic.Dictionary`2[System.String,System.String],Duplicati.Library.Utility.IFilter,System.Int32].invoke_TResult_T1_T2_T3_T4_T5(System.IO.TextWriter,System.Action`1<Duplicati.Library.Main.Controller>,System.Collections.Generic.List`1<string>,System.Collections.Generic.Dictionary`2<string, string>,Duplicati.Library.Utility.IFilter)
  at Duplicati.CommandLine.Program.RunCommandLine (System.IO.TextWriter outwriter, System.IO.TextWriter errwriter, System.Action`1[T] setup, System.String[] args) [0x003ea] in <4f69d9e98e8444e5abbb41ca4fee555e>:0
```

## Steps to reproduce
1. Run `duplicati-cli help advanced` on the commandline

- **Actual result**: Crash
- **Expected result**: Nicely formatted output explaining all advanced cli-options

**Comment by tygill** [comment]
This looks like the string I fixed in #3120. Next canary should fix this.

**Comment by Pectojin** [comment]
I compiled the source from master, containing #3120, and `duplicati-cli help advanced` works as exptected again. 

So yep, will work in 2.0.3.3. I'm closing the issue.
